docs/Attributes.md

```diff
@@ -1,51 +1,91 @@
 # Attributes
 
-Attributes are properties that can be assigned to an entire track or a single detection within a track.  Attributes are created by users utilizing the Settings panel in the main file browser location.  New attributes can be added or existing ones can be modified.
+Attributes are properties that can be assigned to an entire track or a single detection within a track. Attributes are created within a dataset using the TrackDetails panel.
 
 !!! info
-    Attributes are global on the server currently and can be seen or modified by anyone.  There are future plans to move attributes to be specific to projects or users.
+    Attributes were previously global but have been moved to being dataset specific.
 
-## Adding/Editing Attributes
+## Background/Terms
 
-1. On the main browser of VIAME go to the Settings tab
-    1. ![Settings Bar](images/Attributes/SettingsBar.png)
-1. Click on the "+" button to indicate that you want to add a new attribute.
-    1. ![Settings Panel](images/Attributes/SettingsPanel.png)
-1. Provide a descriptive name for the attribute and then determine if it is going to be a track or detection attribute.
-    1. *Track Attributes* - apply data to the entire track
-    1. *Detection Attributes* - apply only to a specific detection on a track
+It’s important to note that there are attribute definitations and then track/detection attributes.  Attributes definitions are all the possible attributes that can be assigned to tracks or detection.  Think of them as a template for the attributes that can be set on tracks/detections.  Track attributes are information for the entire track and detection attributes are information for the specific frame or instance of time.
+
+- **Attribute Definition** - base attribute which defines the name and **type** for either a Track or Detection Attribute
+- **Track Attribute** - attribute with a **value** associated with the entire track
+- **Detection Attribute** - attribute with a **value** specified for a detection during a single frame
+
+Example:
+
+- Attribute Definitions
+    - Track
+        - CompleteTrack: Boolean
+        - FishLength: number (cm)
+    - Detection
+        - Swimming: Boolean
+        - Eating: Boolean
+
+Usage of the above Attribute Definitions for a track and its detections:
+
+- Fish Track 1
+    - Track Attributes
+        - FishLength: 20
+    - Detection Attributes
+        - Frame 1
+            - Eating: true
+        - Frame 2
+            - Swimming: false
+            - Eating: false
+
```

Comment:
```suggestion
Example:

    - Attribute Definitions
        - Track
            - CompleteTrack: Boolean
            - FishLength: number (cm)
        - Detection
            - Swimming: Boolean
            - Eating: Boolean

Usage of the above Attribute Definitions for a track and its detections:

    - Fish Track 1
        - Track Attributes
            - FishLength: 20
        - Detection Attributes
            - Frame 1
                - Eating: true
            - Frame 2
                - Swimming: false
                - Eating: false
```

Breaking this into code blocks made it nicer to look at for me.  Just a thought.  Feel free to ignore.

![Screenshot from 2021-03-15 15-28-57](https://user-images.githubusercontent.com/4214172/111210377-8e63ca00-85a3-11eb-9005-3e56c18829c2.png)

Reply:
that's much better and more compact, I like it.